gen/definitions/nat.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: NAT
+path: Cisco-IOS-XE-native:native/ip/Cisco-IOS-XE-nat:nat
```

Comment:
Isn't NAT something only supported on routers and should therefore have a C9000V test tag?